tests/e2e/app.js

```diff
@@ -0,0 +1,49 @@
+import {CfPaged} from './../../src/index';
+import Page from './../mocks/Page';
+import Vue from 'vue';
+import {FunctionalLimitedCollection, FLC_OVERFLOW_THROW} from './../../node_modules/std-lib/src/FunctionalLimitedCollection'
+import {FunctionalArrayCollection} from './../../node_modules/std-lib/src/FunctionalArrayCollection'
+
+let store = {
+    items: [
+        new Page(1, '<h1>Hello World!</h1>'),
+        new Page(2, '<avatar name="John Doe"></avatar>'),
+    ],
+    selectedItems: []
+};
+
+Vue.component('avatar', {
+    props: ['name'],
+    render(h) {
+        return h('div', [
+            h('h2', [this.name])
+        ])
+    }
+});
+
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Looks like this test cannot be run in a non-browser environment. How can it be changed to work with a simulated DOM?